AsyncDisplayKit/ASPagerNode.m

```diff
@@ -0,0 +1,74 @@
+//
+//  ASPagerNode.m
+//  AsyncDisplayKit
+//
+//  Created by Levi McCallum on 12/7/15.
+//  Copyright © 2015 Facebook. All rights reserved.
+//
+
+#import "ASPagerNode.h"
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+@interface ASPagerNode () <ASCollectionViewDataSource, ASCollectionViewDelegateFlowLayout> {
+  UICollectionViewFlowLayout *_flowLayout;
+}
+
+@end
+
+@implementation ASPagerNode
+
+- (instancetype)init
+{
+  _flowLayout = [[UICollectionViewFlowLayout alloc] init];
+  _flowLayout.scrollDirection = UICollectionViewScrollDirectionHorizontal;
+  _flowLayout.minimumInteritemSpacing = 0;
+  _flowLayout.minimumLineSpacing = 0;
+  
+  self = [super initWithCollectionViewLayout:_flowLayout];
+  if (self != nil) {
+    self.view.asyncDataSource = self;
+    self.view.asyncDelegate = self;
+    
+    self.view.pagingEnabled = YES;
+    self.view.allowsSelection = NO;
+    self.view.showsVerticalScrollIndicator = NO;
+    self.view.showsHorizontalScrollIndicator = NO;
+    
+    ASRangeTuningParameters tuningParams = { .leadingBufferScreenfuls = 1.0, .trailingBufferScreenfuls = 1.0 };
+    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypePreload];
+    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypeRender];
+  }
+  return self;
+}
+
+- (void)reloadData
```

Comment:
Maybe just implement this on ASCollectionNode?  Redeclaring it in the header for clarity so people don't have to look at the superclass is fine for now, but we should make sure that anything relevant to ASCollectionNode is put there.  Same is probably true for setTuningParameters.